bazel/src/test/data/googleapis/google/example/library/v1legacy/BUILD.bazel.baseline

```diff
@@ -0,0 +1,336 @@
+# This file was automatically generated by BuildFileGenerator
```

Comment:
Do we really want to keep the legacy tests? It seems like a lot of text with sophisticated folder structure is added only to test the very small, legacy and rare case of reading from gapic yaml. I think it is ok just to not test that obscure execution path in the tests.

Reply:
Yes - we need to ensure we never break the 10+ clients who still use this because of LTS.